src/main/webapp/app/shared/dashboards/tutor-leaderboard/tutor-leaderboard.component.html

```diff
@@ -41,30 +41,30 @@
                 <td>{{ tutor.numberOfAssessments }}</td>
                 <ng-container *ngIf="!isAtLeastInstructor; else atLeastInstructor">
                     <td>{{ tutor.numberOfAcceptedComplaints }}</td>
-                    <td>{{ tutor.numberOfNotAnsweredMoreFeedbackRequests }}</td>
+                    <td *ngIf="!isExamMode">{{ tutor.numberOfNotAnsweredMoreFeedbackRequests }}</td>
                 </ng-container>
 
                 <ng-template #atLeastInstructor>
-                    <ng-container *ngIf="exercise && course">
+                    <ng-container *ngIf="isExerciseDashboard && exercise && exercise.id && course && course.id">
```

Comment:
```suggestion
                    <ng-container *ngIf="isExerciseDashboard && exercise?.id && course?.id">
```

Reply:
I tried that, but when I build it locally it shows errors, so I did not use it.